src/routes/Detail.js

```diff
@@ -71,6 +72,10 @@ const Detail = () => {
     [auth]
   );
 
+  const copyToClipboard = ({ src }) => {
+    navigator.clipboard.writeText(JSON.stringify(src, null, 2));
+  };
```

Comment:
I don't think we have to keep this as a function, just slap `({ src }) => navigator.clipboard.writeText(JSON.stringify(src, null, 2))` to line 218.